openproblems/tasks/_batch_integration/batch_integration_graph/datasets/lung.py

```diff
@@ -0,0 +1,36 @@
+from .....data.lung import load_lung
+from .....tools.decorators import dataset
+
+
+@dataset(
+    dataset_name="Lung (by batch)",
+    data_url=load_lung.metadata["data_url"],
+    data_reference=load_lung.metadata["data_reference"],
+    dataset_summary="Human pancreatic islet scRNA-seq data from 6 datasets "
+    "across technologies (CEL-seq, CEL-seq2, Smart-seq2, inDrop, Fluidigm C1, "
+    "and SMARTER-seq).",
+    image="openproblems",
+)
+def lung_batch(test=False):
+    import scanpy as sc
+
+    adata = load_lung(test)
+    adata.uns["organism"] = "human"
+    adata.obs["labels"] = adata.obs["cell_type"]
```

Comment:
Please add a comment here that batch is already assigned in the loaded dataset to clarify